runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/HttpBody.kt

```diff
@@ -35,6 +44,14 @@ sealed class HttpBody {
          * Provides [ByteArray] to be sent to peer
          */
         abstract fun bytes(): ByteArray
+
+        override fun deepCopy(): Bytes = object : Bytes() {
```

Comment:
comment

I don't love that we are going to deep copy a payload. I'm not convinced this type should implement `CanDeepCopy`